Makefile

```diff
@@ -3,11 +3,17 @@
 build:
 	./frontend-exec.sh ./build.sh && cargo build
 
+b: build
```

Comment:
I mean `make` without any argument already will run build

Reply:
Maybe it should stay anyway to match the `cargo` aliases?